actions/generate-release-notes/generate-release-notes.sh

```
@@ -129,7 +133,7 @@ github_format_changelog() {
 
 # Build the changelog in JSON format
 json_format_changelog() {
-    local change_log_content="$(printf '%s\n' "${changelog_entries[@]}" | jq -s -c '.')"
+    local change_log_content="$(printf '%s\n' "${changelog_entries[@]:-}" | jq -s -c '.')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)